kernel/arch/dreamcast/kernel/Makefile

```
@@ -18,9 +18,12 @@ COPYOBJS += uname.o
 OBJS = $(COPYOBJS) startup.o
 SUBDIRS =
 
+CRT1_DIR = $(shell $(KOS_CC) $(KOS_SH4_PRECISION) -print-file-name=crt1.o | sed 's/crt1.o//')
+
 myall: $(OBJS)
 	-cp $(COPYOBJS) $(KOS_BASE)/kernel/build/
 	-rm banner.h authors.h banner.o uname.o
+	-cp startup.o $(CRT1_DIR)crt1.o
```

```suggestion
CRT1_PATH = $(shell $(KOS_CC) $(KOS_CFLAGS) -print-file-name=crt1.o)

myall: $(OBJS)
	-cp $(COPYOBJS) $(KOS_BASE)/kernel/build/
	-rm banner.h authors.h banner.o uname.o
	-cp startup.o $(CRT1_PATH)
```

Why remove `crt1.o` with `sed` and then add it back on again?

`$(KOS_CFLAGS)` also would be better than using `$(KOS_SH4_PRECISION)`, just in case the user has other things in their toolchain setup that change the effective multilib path, like endianness.